blog/authors.yml

```diff
@@ -77,3 +77,8 @@ zeke:
   name: zeke
   url: https://github.com/zeke
   image_url: https://github.com/zeke.png?size=96
+
+piotrpdev:
+  name: piotrpdev
+  url: https://github.com/piotrpdev
+  image_url: https://github.com/piotrpdev.png?size=96
```

Comment:
Can you move this to be alphabetically sorted? 🙂

Reply:
Resolved in [`27597cd` (#627)](https://github.com/electron/website/pull/627/commits/27597cd0bc0c3f455a00a2ccf17501237b0ecae3)